fip-0026.md

```diff
@@ -53,13 +51,6 @@ The second scenario, in blue, starts will the seller canceling their listing on
 
 The last scenario, in green, starts with a different user buying a listing on the website.  This second user is denoted as "buyer."  The website then communicates with the smart contract in order to transfer ownership of the address to the buyer and transfer the funds used to buy the domain to the seller.  The marketplace will take a commission for facilitating this transaction.
 
```

Comment:
`Marketplace` is the account owned by the entity running the front end that interacts with this contract. 

The `holder_account` is a designated account to be the intermediary that will hold the domain while listed. 

There was some discussion about it here on the project page too https://fioprotocol.atlassian.net/wiki/spaces/FC/pages/68583579/FIO+Domain+Marketplace

We feel that having the ownership of the domain transferred to a holder account is necessary because it makes sure that when a buyer comes along that it is 100% available for sale. If the ownership doesn't transfer then the seller can still use the domain until it sells, and then, if a buyer comes along it might suddenly, without notice, disappear from their account. 

This is not a good user experience so we decided that transferring is the best way.

we did change the way the transferring of funds will happen. The `buydomain` action will cause 2 transfer actions to happen, one will send to the marketplace account and one will send to the seller so it doesn't need to go through the fio.escrow account at all.

Reply:
Who controls holder_account?

Reply:
The public key is generated and added to the table by msig of the BPs. I think it'd be similar to other smart contracts. 

the marketplace would not control the holder_account.

Reply:
Please add that to the FIP. I think as part of the deployment, the permissions have to be changed to ensure that the holder of the private key cannot transfer all domains out.

Reply:
I modified the terminology line about the holder account to the following:

* **Holder Account** - This is the account that is designated to hold ownership of domains while they are listed for sale. This is set by the BPs via msig and secured in such a way that no one person has access to the account and can transfer the domains away.